src/main/java/net/sf/jabref/exporter/SavePreferences.java

```diff
@@ -51,7 +51,7 @@ public static SavePreferences loadForExportFromPreferences(JabRefPreferences pre
     }
 
     public static SavePreferences loadForSaveFromPreferences(JabRefPreferences preferences) {
-        Boolean saveInOriginalOrder = true;
```

Comment:
That is true. If no explicit sort order is set in the meta data, the entries should be saved in the original order (to modify as little in the file as possible). 

As far as I can see there is currently no test for that. Could you perhaps add one in this PR?